### PR TITLE
Added more questions and answers to FAQ section, new :doc:`OpenGL FAQ </faq/opengl>`

### PR DESCRIPTION
**Pull request by behackl**
## Overview: What does this pull request change?

See title.

Link to documentation:
- [FAQ: general usage](https://manimce--2767.org.readthedocs.build/en/2767/faq/general.html)
- [FAQ: OpenGL rendering](https://manimce--2767.org.readthedocs.build/en/2767/faq/opengl.html)

### CODE REVIEW

File: docs/source/faq/opengl.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+or specific OpenGL classes like `OpenGLSurface`, but documentation for some of
+them is available in form of docstrings
+[in the source code](https://github.com/ManimCommunity/manim/tree/main/manim/mobject/opengl).
+
+Furthermore, [this user guide by *aquabeam*](https://www.aquabeam.me/manim/opengl_guide/)
+can be helpful to get started using the OpenGL renderer.
+
+---
+
+### I am trying to run an interactive scene with `--renderer=opengl` and `Scene.interactive_embed`, but an error (`sqlite3.ProgrammingError`) is raised. How can I fix this?
```
Comment (naveen521kk) [inline docs/source/faq/opengl.md:16]:
```suggestion
## I am trying to run an interactive scene with `--renderer=opengl` and `Scene.interactive_embed`, but an error (`sqlite3.ProgrammingError`) is raised. How can I fix this?
```

**Comment by behackl** [comment]
Applied suggested changes, thanks!

File: docs/source/faq/general.md [hunk 2]
```diff
@@ -31,6 +31,44 @@ Manim will not call your method and simply output a black frame.
 
 ---
 
+## What are the default measurements for Manim's scene?
+
+The scene measures 8 units in height and has a default ratio of 16:9,
+which means that it measures :math:`8 \cdot 16 / 9 = 14 + 2/9` units in width.
```
Comment (naveen521kk) [inline docs/source/faq/general.md:37]: This doesn't render for some reason: https://manimce--2767.org.readthedocs.build/en/2767/faq/general.html#what-are-the-default-measurements-for-manim-s-scene

Comment (behackl) [reply docs/source/faq/general.md:37]: I forgot that I am writing markdown, not ReST.

```suggestion
which means that it measures {math}`8 \cdot 16 / 9 = 14 + 2/9` units in width.
```

**Review by naveen521kk — APPROVED** [review]
lgtm

**Pull request merged**